nltk/test/unit/test_seekable_unicode_stream_reader.py

```diff
@@ -96,41 +103,11 @@ def test_reader():
                 pass
 
 
-# nose shows the whole string arguments in a verbose mode; this is annoying,
-# so large string test is separated.
-
-LARGE_STRING = (
-    """\
-This is a larger file.  It has some lines that are longer \
-than 72 characters.  It's got lots of repetition.  Here's \
-some unicode chars: \xee \u0123 \uffe3 \ueeee \u2345
-
-How fun!  Let's repeat it twenty times.
-"""
-    * 10
-)
-
-
-def test_reader_on_large_string():
-    for encoding in ENCODINGS:
-        try:
-            # skip strings that can't be encoded with the current encoding
-            LARGE_STRING.encode(encoding)
-
-            def _check(encoding, n=1000):
-                check_reader(LARGE_STRING, encoding, n)
-
-            _check(encoding)
-
-        except UnicodeEncodeError:
-            pass
-
-
 def test_reader_stream_is_closed():
     reader = SeekableUnicodeStreamReader(BytesIO(b''), 'ascii')
-    assert reader.stream.closed is False
+    assert reader.stream.closed == False
```

Comment:
Actually `is` is preferred here. `False` and `True` are singletons so you can (and should, for better readability) check them by identity.

Another option is to say this:

```python
assert not reader.stream.closed
```

Reply:
didn't know about them being singletons. looks like the [python style guide](https://www.python.org/dev/peps/pep-0008/) suggests doing neither `==` nor `is`. Wow, TIL!